output/mod_resultScope.f90

```diff
@@ -0,0 +1,950 @@
+!----------------------------------------------------------------------
```

Comment:
How is this file going to be compiled and when?

Reply:
was a stray file, I have deleted it